sphinx/builders/_epub_base.py

```diff
@@ -418,11 +418,10 @@ def copy_image_files_pil(self) -> None:
                     logger.warning(__('cannot copy image file %r: %s'),
                                    path.join(self.srcdir, src), err)
                 continue
-            if self.config.epub_fix_images:
-                if img.mode in ('P',):
-                    # See the Pillow documentation for Image.convert()
-                    # https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.convert
-                    img = img.convert()
+            if self.config.epub_fix_images and img.mode in ('P',):
```

Comment:
@jakobandersen is there a reason you don't like collapsing the statements [here](https://github.com/sphinx-doc/sphinx/pull/10077/files#r780977852), but don't mind collapsing them at this line? As an outsider, I'm struggling to see the semantic difference. Maybe you can give me a little guidance as to your reasoning?

Reply:
As mentioned originally: I only reviewed the C and C++ domains as that's the code I know best. The example you linked to is not the most problematic, and I could live with that particular change. I don't know the epub format, so I can't comment on what is reasonable here.